test/root_test.rb

```diff
@@ -4,17 +4,43 @@ module AdequateSerializer
   class RootTest < Minitest::Spec
     def test_default_root
       peggy = Person.new(id: 1, name: 'Peggy')
+
       TestSerializer.new(peggy).as_json.keys.must_equal [:person]
     end
 
+    def test_default_root_of_array
+      peggy = Person.new(id: 1, name: 'Peggy')
+
+      Collection.new([peggy]).as_json.keys.must_equal [:people]
+    end
+
+    def test_default_root_of_empty_array
+      Collection.new([]).as_json.keys.must_equal [:nil]
+    end
+
+    def test_default_root_of_relation
+      peggy = Person.new(id: 1, name: 'Peggy')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.